NEWS.md

```diff
@@ -1,8 +1,23 @@
-# scoringutils 1.2.2
+# scoringutils 1.3
 
-This minor update addresses comments made by review from the Journal of Statistical Software (see preprint of the manuscript [here](https://arxiv.org/abs/2205.07090)).
+This major update introduces a lot of breaking changes and addresses comments made by review from the Journal of Statistical Software (see preprint of the manuscript [here](https://arxiv.org/abs/2205.07090)).
```

Comment:
Suggest leading with addresses review comments and improves user interface and then into their being breaking changes and what to do if you need the old verrsion

Reply:
Done ✅